examples/gio_task/file_size/mod.rs

```diff
@@ -8,6 +8,8 @@ glib::wrapper! {
     pub struct FileSize(ObjectSubclass<imp::FileSize>);
 }
 
+unsafe impl Send for FileSize {}
```

Comment:
```suggestion
unsafe impl Send for FileSize {}
unsafe impl Sync for FileSize {}
```

Either a `GObject` is both or neither because of refcounting.

Is `imp::FileSize` both? Otherwise this would be unsound.